java/test/jmri/jmrix/anyma/UsbLightManagerTest.java

```diff
@@ -0,0 +1,37 @@
+package jmri.jmrix.anyma;
+
+import jmri.util.JUnitUtil;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Tests for UsbLightManager class.
+ *
+ * @author George Warner Copyright (C) 2017
+ * @since 4.9.6
+ */
+public class UsbLightManagerTest {
```

Comment:
This test needs to extend jmri.managers.AbstractLightMgrTestBase

Reply:
Done!